underdocs-error/src/main/kotlin/underdocs/error/pretty/DefaultErrorPrettyPrinter.kt

```diff
@@ -1,7 +1,40 @@
 package underdocs.error.pretty
 
+import underdocs.error.BaseUnderdocsError
+
 class DefaultErrorPrettyPrinter : ErrorPrettyPrinter {
     override fun print(exception: Exception): String {
-        TODO("Not yet implemented")
+        var prettyPrintedError = """"""
+
+        if (exception is BaseUnderdocsError) {
```

Comment:
In Kotlin, is this the correct way to check if the exception inherits from the Base error class?

Reply:
Sure it is!